docs/src/pages/index.tsx

```diff
@@ -92,74 +112,80 @@ export default function Home(
+          Create beautiful Loops that seamlessly integrated with Olive Helps.
+        </p>
+        <p className={styles.sectionDescription}>
+          <a href="" className={styles.buttonInversePadding}>
+            View the design system
+          </a>
         </p>
-        <div className={styles.aptitudeList}>{aptitudeItems}</div>
       </Section>
       <Section sectionClassName={styles.sectionBackground}>
-        <h2 className={styles.sectionTitle}>Read the Guides</h2>
-        <p>We have guides available to help you at various steps.</p>
-        <div className={styles.aptitudeList}>{guideItems}</div>
+        <h2 className={styles.sectionTitle}>Additional Resources</h2>
+        <div className={styles.guideList}>{guideItems}</div>
+        <div>
+          <h3>Need help getting started?</h3>
+          <p>
+            Submit a request or email your Olive Helps developer contact for further assistance.
+          </p>
+          <a href="">Submit a request</a>
+        </div>
       </Section>
       <Section sectionClassName={styles.sectionBackground}>
-        <h2 className={styles.sectionTitle}>Getting Help</h2>
         <p>
-          Our Olive Helps developers are here to help you succeed! Visit{' '}
           <a
-            href="https://github.com/open-olive/loop-development-kit/issues"
+            className={styles.termsLink}
+            href="https://olive.page.link/olive-helps-terms"
             target="_blank"
-            rel="noopener"
+            rel="noopener noreferrer"
           >
-            this page
-          </a>{' '}
-          to submit an issue or request, or email your Olive Helps developer contact if you need any
-          additional help.
-        </p>
-      </Section>
-      <Section sectionClassName={styles.sectionBackground}>
-        <p>
+            About Olive
+          </a>
+
+          <a
+            className={styles.termsLink}
+            href="https://olive.page.link/olive-helps-terms"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Terms and Conditions
+          </a>
           <a
             className={styles.licenseLink}
             href="https://github.com/open-olive/loop-development-kit/blob/main/LICENSE"
             target="_blank"
             rel="noopener noreferrer"
           >
-            Loop Development Kit Open Source License
+            LDK Open Source License
+          </a>
+          <a href="https://www.instagram.com/oliveai__/" target="_blank" rel="noopener noreferrer">
+            INSTAGRAM ICON HERE
           </a>
-          {'|'}
           <a
-            className={styles.termsLink}
-            href="https://olive.page.link/olive-helps-terms"
+            href="https://www.linkedin.com/company/oliveai"
             target="_blank"
             rel="noopener noreferrer"
           >
-            Olive Helps Terms &amp; Conditions
+            LINKEDIN ICON HERE
```

Comment:
Is this intended?

Reply:
For the moment! This is a partial PR review - I haven't gotten to that section yet.